Sources/PostgresNIO/New/Messages/RowDescription.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 11.01.21.
+//
+
+extension PSQLBackendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLBackendMessage {
```